.changeset/funny-chicken-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-pwa/composables-next": minor
+---
+
+Set user details when fully active
```

Comment:
I'm not sure what should I understand (as a user) from this changelog entry:
- in which composable?
- what does it mean?
- what impact does it have on my code?

is this really a feature or bugfix?

Reply:
Because we do not have a version in relation to the backend, this is a feature. We are implementing a new flow.
Changed